saleor/warehouse/tasks.py

```diff
@@ -11,3 +13,28 @@ def delete_empty_allocations_task():
     count, _ = Allocation.objects.filter(quantity_allocated=0).delete()
     if count:
         task_logger.debug("Removed %s allocations", count)
+
+
+@app.task
+def update_stocks_quantity_allocated_task():
+    count = 0
+    corrected_count = 0
+    for stock in Stock.objects.iterator():
+        count += 1
+        quantity_allocated = stock.allocations.aggregate(
+            quantity_allocated=Coalesce(Sum("quantity_allocated"), 0)
+        )["quantity_allocated"]
+        if stock.quantity_allocated != quantity_allocated:
+            task_logger.info(
+                "Mismatch updating quantity_allocated: stock {} had "
+                "{} allocated, but should have {}.".format(
+                    stock.pk, stock.quantity_allocated, quantity_allocated
+                )
```

Comment:
We shouldn't use f'strings and format in the logs, we should pass values as args of the logger. 
```suggestion
                "Mismatch updating quantity_allocated: stock %d had "
                "%d allocated, but should have %d.",
                 stock.pk, stock.quantity_allocated, quantity_allocated
```